mgmtsystem_survey/data/survey_survey_data.xml

```diff
@@ -0,0 +1,19 @@
+<openerp>
```

Comment:
I thought it would be a good practice to provide the basis, like the list of root causes or NC origins.